test/file_input_tests/MIDI_file_class_test.py

```diff
@@ -0,0 +1,58 @@
+
+import src.file_input.MIDI_file_class as MIDI_FC
+
+class TestParse:
+
+    def test_parse_midi_file(self):
+        """ This is the test to ensure the function returns the correct list of midi messages.
+        TODO: complete this test, test with smaller file
```

Comment:
Fine with leaving this incomplete for now, but make sure to put something about this in Trello so it doesn't get lost

Reply:
Good idea, my thinking is that we will go back during our testing phase and I can test it.